workflow/training/whisper.py

```
@@ -35,174 +29,133 @@
-                print("Number of rows:", len(split_data))
-                
+        
         if "audio" not in train_dataset.column_names:
             audio_column = train_dataset.column_names[0]
             train_dataset = train_dataset.rename_column(audio_column, "audio")
+            logger.info(f"Renamed column to 'audio'")
 
         if "sentence" not in train_dataset.column_names:
             sentence_column = train_dataset.column_names[1]
             train_dataset = train_dataset.rename_column(sentence_column, "sentence")
+            logger.info(f"Renamed column to 'sentence'")
```

Ruff

f-string without any placeholders

Remove extraneous `f` prefix

(F541)